### ISSUE TITLE
show the next "rank" 

### ISSUE THREAD

**Issue by fredao** [streetcomplete/StreetComplete#4561]
**Use case**

For motivation, it would be nice to see how many "points" the next higher rank has. 

**Proposed Solution**

Maybe something like this
https://qu.ax/KVxj.jpeg

Next level
Next rank ... or whatever

More or less like the leaderboard in MapRoulette
![grafik](https://user-images.githubusercontent.com/1868849/198258763-c7d22c97-1ce0-4f4f-a6fa-3a84026a9182.png)

**Comment by mnalis** [comment]
Related FAQ entry: ["I like gamification factor of StreetComplete, and have an idea how to increase it even more"](https://wiki.openstreetmap.org/wiki/StreetComplete/FAQ#I_like_gamification_factor_of_StreetComplete,_and_have_an_idea_how_to_increase_it_even_more)

**Comment by fredao** [comment]
I am well aware of the problem with gamification. 

The leaderboard of MapRoulette was only an example, as I did not know whether my request was understood. The example should not be realised under any circumstances. 

If so, then there should only be a small hint as indicated in the picture.  https://qu.ax/KVxj.jpeg 

The idea came from my granddaughter because she wanted to know how far away the next step is.

**Comment by mnalis** [comment]
![small_KVxj](https://user-images.githubusercontent.com/156656/198300966-7edae805-2acd-4dcf-a6d5-f41338d8509e.jpeg) (added suggested https://qu.ax/KVxj.jpeg picture inline)

**Comment by fredao** [comment]
@mnalis if you think a small number is to much gamification, one should rethink My Profile - by type, by country and all the awards ;-)

**Comment by mnalis** [comment]
> @mnalis if you think a small number is to much gamification,

I'm actually undecided about this specific suggestion, that's why I wanted to hear opinions of others... 
(It's not up to me to decide whether something like it would be accepted or not anyway :smiley:)

> one should rethink My Profile - by type, by country and all the awards

those are probably not problematic...

<details>


</details>

---

This proposal specifically (if implemented) might turn gamification concern up quite a notch, as it is also in (most troublesome) competing-with-other-humans category, and it gives more incentive to "cheat" (as you can see how close you are to getting to getting new dopamine hit, which current `global/country rank` don't). On the plus side, this proposal does not show any _names_, which would be _sure way_ to cause problems. 

On negative feedback side (as mentioned before) there is also  possible issue that (if always shown!) it **can be quite demotivating too** if one sees there is huge amount of work needed to advance to next level (see for example also https://github.com/streetcomplete/StreetComplete/issues/3230#issuecomment-908682445)

**Comment by retiolus** [comment]
To add my little grain of sand to this debate, and I agree with everything that has been said by @mnalis I also think that in some cases it can be demotivating to make editions for a long time and not see our ranking increase (which is sincerely never the case).

And in reality, even at a high ranking (I'm talking in my case where I'm at #675 world level), even doing 50 editions, I still go up several points in the ranking at once. So I think that the "it can be demotivating to see that you are very far from the next ranking" only applies to a minority of people who are very high in this ranking.

**Comment by YoshiRulz** [comment]
How about showing progress for achievements, maybe only when you're close?

**Comment by mnalis** [comment]
> [...] even doing 50 editions, I still go up several points in the ranking at once. So I think that the "it can be demotivating to see that you are very far from the next ranking" only applies to a minority of people who are very high in this ranking.

This is true, it gets slower only when you get in top-few-hunderds or so, but then those top users are not really your main targets audience as you note, and there is by a definition a small number of top-performers anyway... 

However to address second part of the equation: while `just 50` quests is not a problem to someone in urban city centre (like me and I guess you too) who could solve that in one easy afternoon walk, it might be quite more time-consuming if someone lives/commutes in more rural areas (even with resurvey preference set to `ask more often`) where quests are much much rarer. 

But I agree with @matkoniecz, let's see how much extra life [v49.0-beta1](https://github.com/streetcomplete/StreetComplete/discussions/4573) changes to award screens bring us first!
